### PR TITLE
fix channels_first format

### PR DESCRIPTION
**Pull request by SergeyTsimfer**
Moved build_configs from childs of Inception_base to the parent class itself

### CODE REVIEW

File: batchflow/models/tf/inception_base.py [hunk 1]
```diff
@@ -32,6 +32,19 @@ class Inception(TFModel):
             value : dict
                 specific parameters (e.g. filters)
     """
+
+    def build_config(self, names=None):
```
Comment (AlexeyKozhevin) [inline batchflow/models/tf/inception_base.py]: Add docstring

File: batchflow/models/tf/inception_base.py [hunk 2]
```diff
@@ -32,6 +32,19 @@ class Inception(TFModel):
...
+        config = super().build_config(names)
+        if config.get('head/units') is None:
+            config['head/units'] = self.num_classes('targets')
+        if config.get('head/filters') is None:
+            config['head/filters'] = self.num_classes('targets')
+
+        data_format = config['data_format']
+        config['initial_block'].update({'data_format':data_format})
```
Comment (AlexeyKozhevin) [inline batchflow/models/tf/inception_base.py]: Add whitespace before `data_format` and check your code by pylint

File: batchflow/models/tf/inception_base.py [hunk 3]
```diff
@@ -32,6 +32,19 @@ class Inception(TFModel):
...
+        config['body'].update({'data_format':data_format})
```
Comment (AlexeyKozhevin) [inline batchflow/models/tf/inception_base.py]: The same

File: batchflow/models/tf/inception_resnet_v2.py [hunk 4]
```diff
@@ -43,14 +43,6 @@ def default_config(cls):
...
-        config = super().build_config(names)
-        if config.get('head/units') is None:
-            config['head/units'] = self.num_classes('targets')
-        if config.get('head/filters') is None:
-            config['head/filters'] = self.num_classes('targets')
-        return config
-
     @classmethod
     def initial_block(cls, inputs, name='initial_block', **kwargs):
         """Input network block.
```
Comment (AlexeyKozhevin) [inline batchflow/models/tf/inception_resnet_v2.py]: Add whitespace before `Input`

File: batchflow/models/tf/squeezenet.py [hunk 5]
```diff
@@ -53,6 +53,10 @@ def build_config(self, names=None):
             config['head/units'] = self.num_classes('targets')
         if config.get('head/filters') is None:
             config['head/filters'] = self.num_classes('targets')
+
+        data_format = config['data_format']
```
Comment (AlexeyKozhevin) [inline batchflow/models/tf/squeezenet.py]: Are you sure that `config` has `data_format`?

File: batchflow/models/tf/unet.py [hunk 6]
```diff
@@ -64,6 +64,11 @@ def build_config(self, names=None):
             config['body/filters'] = (2 ** np.arange(config['body/num_blocks']) * 64).tolist()
         if config.get('head/num_classes') is None:
             config['head/num_classes'] = self.num_classes('targets')
+
+        data_format = config['data_format']
```
Comment (AlexeyKozhevin) [inline batchflow/models/tf/unet.py]: The same comments (pylint, missed key...)

Comment (AlexeyKozhevin) [reply batchflow/models/tf/unet.py]: Check it everywhere

File: batchflow/models/tf/densenet.py [hunk 7]
```diff
@@ -70,6 +69,11 @@ def build_config(self, names=None):
             config['head/units'] = self.num_classes('targets')
         if config.get('head/filters') is None:
             config['head/filters'] = self.num_classes('targets')
+
+        data_format = config.get('data_format')
+        if data_format:
+            config['initial_block'].update({'data_format': data_format})
+            config['body'].update({'data_format': data_format})
```
Comment (roman-kh) [inline batchflow/models/tf/densenet.py]: This is not needed as ``data_format`` is passed via ``common`` block

File: batchflow/models/tf/inception_base.py (hunk 1 above)
Comment (SergeyTsimfer) [reply batchflow/models/tf/inception_base.py]: Is that really needed? There are no docstrings for `build_config` in any of the models presented.

**Pull request closed without merging**